backend/app/controllers/uoas.js

```diff
@@ -66,23 +67,52 @@ module.exports = {
         });
     },
 
-    insertOne: function (req, res, next) {
+    insert: function (req, res, next) {
         var thunkQuery = req.thunkQuery;
+        var uoas = req.body.name.split(/\s*,\s*/);
+        var sqlString = "'"+uoas.toString().replace(/'/g, "''").replace(/,/g, "','")+"'";
```

Comment:
I am thinking... would it be better do these operations on the front end and then send a list back to the server? I feel like something could easily go wrong here and we would probably want to catch that before we even get to the server. This function shouldn't really be doing any manipulations to the data it receives.

Reply:
For the `sqlString`, it's necessary because of how Postgres handles quotes (and the frontend shouldn't really know that it's dealing with whatever database).